storage/src/backend/connection.rs

```diff
@@ -452,6 +449,7 @@ impl Connection {
                             }
                             let m = &self.mirror_state.mirrors[idx];
                             if m.status.load(Ordering::Relaxed) {
+                                warn!("mirror server has been changed to {:?}", m);
```

Comment:
We need to standardize the lowercase of the message text.

Reply:
Fixed.